doc/controllers_index.rst

```diff
@@ -26,24 +26,27 @@ The following standard controllers are implemented:
   - `Joint Trajectory Controller <joint_trajectory_controller/docs/index.rst>`_ - provided a list of waypoints or target point defined with position, velocity and acceleration, the controller interpolates joint trajectories through it.
   - ... <the list is not complete> ...
 
-
-
-
 .. _common hardware interface definitions: https://github.com/ros-controls/ros2_control/blob/master/hardware_interface/include/hardware_interface/types/hardware_interface_type_values.hpp
```

Comment:
Formatting here looks funky on a couple different markdown viewers I tried. Suggest making it a new section:

```suggestion
Common Hardware Interface Definitions
--------------
https://github.com/ros-controls/ros2_control/blob/master/hardware_interface/include/hardware_interface/types/hardware_interface_type_values.hpp
```

Reply:
This is rst, so it should be OK there. Or did you try also the RestructuredText viewer?

Reply:
Ah, I see. So this is basically a link. That looks good, then.